pymc3/tests/test_step.py

```diff
@@ -3,16 +3,20 @@
 
 from .checks import close_to
 from .models import simple_categorical, mv_simple, mv_simple_discrete, simple_2model
+from .helpers import SeededTest
+from pymc3 import df_summary, traceplot
 from pymc3.sampling import assign_step_methods, sample
 from pymc3.model import Model
 from pymc3.step_methods import (NUTS, BinaryGibbsMetropolis, CategoricalGibbsMetropolis,
                                 Metropolis, Slice, CompoundStep,
                                 MultivariateNormalProposal, HamiltonianMC)
-from pymc3.distributions import Binomial, Normal, Bernoulli, Categorical
+from pymc3.distributions import Binomial, Normal, Bernoulli, Categorical, Uniform
 
 from numpy.testing import assert_array_almost_equal
+from nose_parameterized import parameterized
```

Comment:
Added it to the dependencies, for future use even though I'm probably not going to use it for this. Perhaps we should remove it though